cmd/sqlcmd/sqlcmd_test.go

```diff
@@ -90,15 +74,23 @@ func TestValidCommandLineToArgsConversion(t *testing.T) {
 		{[]string{"--version"}, func(args SQLCmdArguments) bool {
 			return args.Version
 		}},
-		{[]string{"-s", "|", "-w", "10", "-W"}, func(args SQLCmdArguments) bool {
-			return args.TrimSpaces && args.ColumnSeparator == "|" && *args.ScreenWidth == 10
-		}},
 	}
 
 	for _, test := range commands {
 		arguments := &SQLCmdArguments{}
-		parser := newKong(t, arguments)
-		_, err := parser.Parse(test.commandLine)
+		cmd := &cobra.Command{
+			Use:   "testCommand",
+			Short: "A brief description of my command",
+			Long:  "A long description of my command",
+			Run: func(cmd *cobra.Command, argss []string) {
+				// Command logic goes here
+				vars := sqlcmd.InitializeVariables(!args.DisableCmdAndWarn)
+				setVars(vars, &args)
+			},
+		}
+		setFlags(cmd, arguments)
+		cmd.SetArgs(test.commandLine)
+		err := cmd.Execute()
```

Comment:
>	err := cmd.Execute() [](http://example.com/codeflow?start=1&length=21)

we can't unit test Execute because it is going to try to do everything.
Instead make sure the function that populates SQLCmdArguments is individually callable and unit test that function